docs/utils.md

```diff
@@ -234,3 +234,35 @@ $ freqtrade -c config_binance.json list-pairs --all --base BTC ETH --quote USDT
 ```
 $ freqtrade list-markets --exchange kraken --all
 ```
+
+## Test pairlist
+
+Use the `test-pairlist` subcommand to test the configuration of [dynamic pairlists](configuration.md#pairlists).
+
+Requires a configuration with specified `pairlists` attribute.
+Can be used to generate static pairlists to be used during backtesting / hyperopt.
+
+### Examples
```

Comment:
examples should probably be below usage block

Reply:
I'm not sure ... why exactly? 
examples are most of the time more important as they show you how to "really" use the command for certain cases.

To me, the "full usage"  block is just there for completeness - for many commands which have many options (list-pairs for example, but there are others, too) the examples are better / more useful- since they show you how to combine the options - and don't just make you read the help strings ...

Reply:
1) because you first need to give the user some introduction of all options available before giving examples. Have a look at the man pages -- they first give the list of all options, then their extended descriptions (examples can be inserted there as well) and only then some more examples are placed, when the reader have knowledge of all the options
2) this is how info is placed in the Start the Bot and Plotting parts -- some general introduction (with probably a general example, that example has no target to describe the options, just to give some general understanding about the command), then the list of options and the examples follow then. The page on `new-strategy` should also be reformatted this way, to have examples after the list of options.